workflows/sample_workflow.ts

```diff
@@ -0,0 +1,52 @@
+import { DefineWorkflow, Schema } from "deno-slack-sdk/mod.ts";
+import { SampleFunctionDefinition } from "../functions/sample_function.ts";
+
+const SampleWorkflow = DefineWorkflow({
+  callback_id: "sample_workflow",
+  title: "Sample workflow",
+  description: "A sample workflow",
+  input_parameters: {
+    properties: {
+      interactivity: {
+        type: Schema.slack.types.interactivity,
+      },
+      channel: {
+        type: Schema.slack.types.channel_id,
+      },
+    },
+    required: ["interactivity"],
+  },
+});
+
+const inputForm = SampleWorkflow.addStep(
```

Comment:
Should leave some comments in this file. Like on line 20 here we could say something like "If your workflow requires collecting input from end users, we recommend using the openForm built-in function as a first step"

Reply:
In complete agreement: we should add comments. I'll take a pass at it!

Reply:
Added comments to each component that aligns with the content we have in the README. I think it makes it much more usable and educational. Thanks for the suggestion!